.evergreen/pkgconfig-tests.sh

```diff
@@ -60,92 +62,105 @@ run_cmake -DUSE_SHARED_LIBBSON=OFF \
 
 export PKG_CONFIG_PATH
 PKG_CONFIG_PATH="$mongocrypt_pkg_config_path:$libbson_pkg_config_path"
 
-echo "Validating pkg-config scripts"
+echo "Validating pkg-config scripts ..."
 pkg-config --debug --print-errors --exists libmongocrypt-static
 pkg-config --debug --print-errors --exists libmongocrypt
+echo "Validating pkg-config scripts ... done"
 
-# Build example-state-machine, static linking against libmongocrypt
+echo "Build example-state-machine, static linking against libmongocrypt ..."
 gcc $(pkg-config --cflags libmongocrypt-static libbson-static-1.0) \
     -o "$pkgconfig_tests_root/example-state-machine" \
     "$LIBMONGOCRYPT_DIR/test/example-state-machine.c" \
     $(pkg-config --libs libmongocrypt-static)
 run_chdir "$LIBMONGOCRYPT_DIR" "$pkgconfig_tests_root/example-state-machine"
+echo "Build example-state-machine, static linking against libmongocrypt ... done"
 
-# Build example-no-bson, static linking against libmongocrypt
+echo "Build example-no-bson, static linking against libmongocrypt ..."
 gcc $(pkg-config --cflags libmongocrypt-static) \
     -o "$pkgconfig_tests_root/example-no-bson" \
     "$LIBMONGOCRYPT_DIR/test/example-no-bson.c" \
     $(pkg-config --libs libmongocrypt-static)
 command "$pkgconfig_tests_root/example-no-bson"
+echo "Build example-no-bson, static linking against libmongocrypt ... done"
 
-# Build example-state-machine, dynamic linking against libmongocrypt
+echo "Build example-state-machine, dynamic linking against libmongocrypt ..."
 gcc $(pkg-config --cflags libmongocrypt libbson-static-1.0) \
     -o "$pkgconfig_tests_root/example-state-machine" \
     "$LIBMONGOCRYPT_DIR/test/example-state-machine.c" \
     $(pkg-config --libs libmongocrypt)
 run_chdir "$LIBMONGOCRYPT_DIR" \
     env LD_LIBRARY_PATH="$mongocrypt_install_dir/lib:$mongocrypt_install_dir/lib64" \
     "$pkgconfig_tests_root/example-state-machine"
+echo "Build example-state-machine, dynamic linking against libmongocrypt ... done"
 
-# Build example-no-bson, dynamic linking against libmongocrypt
+echo "Build example-no-bson, dynamic linking against libmongocrypt ..."
 gcc $(pkg-config --cflags libmongocrypt) \
     -o "$pkgconfig_tests_root/example-no-bson" \
     "$LIBMONGOCRYPT_DIR/test/example-no-bson.c" \
     $(pkg-config --libs libmongocrypt)
 env LD_LIBRARY_PATH="$mongocrypt_install_dir/lib:$mongocrypt_install_dir/lib64" \
     "$pkgconfig_tests_root/example-no-bson"
+echo "Build example-no-bson, dynamic linking against libmongocrypt ... done"
 
 # Clean up prior to next execution
 rm -r "$mongocrypt_install_dir"
 
-# Build libmongocrypt, dynamic linking against libbson
+echo "Build libmongocrypt, dynamic linking against libbson ..."
+# Enable extra alignment on imported libbson to match installed libbson.
 run_cmake -DUSE_SHARED_LIBBSON=ON \
        -DENABLE_BUILD_FOR_PPA=OFF \
+       -DENABLE_EXTRA_ALIGNMENT=ON \
```

Comment:
Do we have more information about the "installed libbson" library? Can it be updated so it is also configured with `ENABLE_EXTRA_ALIGNMENT=OFF`?

Reply:
Yes the "installed libbson" is installed within this script. Updated the "installed libbson" to set `ENABLE_EXTRA_ALIGNMENT=OFF` to match recommended libbson default.